services/scan/src/store.ts

```diff
@@ -57,7 +58,7 @@ export enum ConfigKey {
   MarkThresholdOverrides = 'markThresholdOverrides',
   // @deprecated
   SkipElectionHashCheck = 'skipElectionHashCheck',
-  CurrentPrecinctId = 'currentPrecinctId',
+  PrecinctSelectionKey = 'precinctSelection',
```

Comment:
For consistency with the other keys in this enum, we may want to shorten this to `PrecinctSelection = 'precinctSelection'`. The `Key` in `ConfigKey` should make it clear that this is a key, e.g. `ConfigKey.PrecinctSelection`

Reply:
I tried this but there's a variable name conflict in the file if you do that, will take another look to see if there's another way.

Reply:
Renamed the key by importing the type as:
```
  PrecinctSelection as PrecinctSelectionType,
```